redash/tasks/queries.py

```diff
@@ -278,20 +278,30 @@ class QueryExecutionError(Exception):
     pass
 
 
+def _resolve_user(user_id, is_api_key):
+    if user_id is not None:
+        if is_api_key:
+            api_key = user_id
+            q = models.Query.query.filter(models.Query.api_key == api_key).one()
+            return models.ApiUser(api_key, q.org, q.groups)
+        else:
+            return models.User.query.get(user_id)
```

Comment:
Same here (we probably have a `#get_by_id` or `#get_by_id_and_org` on this model anyway).

Reply:
34404b44